component/async/sqs/sqs.go

```diff
@@ -218,12 +215,12 @@ func (c *consumer) Consume(ctx context.Context) (<-chan async.Message, <-chan er
 			if sqsCtx.Err() != nil {
 				return
 			}
-			log.Debugf("polling SQS queue %s for messages", c.queueName)
+			log.Debugf("Consume: polling SQS queue %s for %d messages", c.queueName, *c.maxMessages)
```

Comment:
a Go convention is not to have uppercase logs and error messages.

Reply:
Applied in next commit